SR2023/2023-02-23-virtual-competition-reminder.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Virtual Competition and Tech Day Reminder
+---
+
+Hi
+
+We're just reminding you about the [virtual competition](https://studentrobotics.org/events/sr2023/virtual-competition/) happening this weekend.
+
+[Submit your robot code](https://studentrobotics.org/code-submitter/).
```

Comment:
```suggestion
[Submit your robot code](https://studentrobotics.org/code-submitter/) by 8pm on Friday.
```

Reply:
UK time?

Reply:
The full detail (date & time) is included in the following paragraph, so I wasn't sure we needed it here as well. I don't feel strongly though.

Reply:
As it's in the next paragraph, I think it's ok to leave out here.

Reply:
The timezone or the time entirely? My thought was to make it explicit what the deadline was.

Reply:
We mention the time and the timezone 2 paragraphs later, I don't think we need it in this main CTA. Alternatively, we inline the link into that paragraph and reflow the intermediary paragraph into it